app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -316,6 +317,7 @@ enum class KeyboardLayout(
     ENSVThumbKeyProgramming(KB_EN_SV_THUMBKEY_PROGRAMMING),
     ESThumbKeySymbols(KB_ES_THUMBKEY_SYMBOLS),
     ENThumbKeyWords(KB_EN_THUMBKEY_WORDS),
+    ENThumbKeyWordsSymbols(KB_EN_THUMBKEY_WORDS_SYMBOLS),
```

Comment:
Read the comment at the top of this function. Also in the future, look at other PRs for examples.

Reply:
Done. Sorry for so much churn, I definitely should've looked closer.